packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton.doc.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { VisuallyHidden } from '~components/VisuallyHidden'
+import { Icon } from '~components/__future__'
+import { LinkButton } from '../index'
+
+const meta = {
+  title: 'Actions/LinkButton/LinkButton (v3)',
+  component: LinkButton,
+  args: {
+    children: 'Label',
+    href: '#link-button-clicked',
+  },
+  argTypes: {
+    icon: {
+      options: ['delete', 'arrow', 'plus'],
+      mapping: {
+        delete: <Icon isPresentational name="delete" />,
+        arrow: <Icon isPresentational name="arrow_forward" />,
+        add: <Icon isPresentational name="add" />,
+      },
+    },
+  },
+} satisfies Meta<typeof LinkButton>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {}
+
+export const LinkButtonOpensInNewTab: Story = {
+  args: {
+    children: (
+      <>
+        Label
+        <VisuallyHidden> opens a new tab</VisuallyHidden>
+      </>
+    ),
+    href: 'https://www.google.com',
+    target: '_blank',
+    rel: 'noopener noreferrer',
+    icon: <Icon isPresentational name="open_in_new" shouldMirrorInRTL />,
+    iconPosition: 'end',
+  },
+}
+
+export const LinkButtonVariants: Story = {
+  render: ({ children: _, ...otherArgs }) => (
+    <>
+      <LinkButton {...otherArgs} variant="primary">
+        Primary
+      </LinkButton>
+      <LinkButton {...otherArgs} variant="secondary">
+        Secondary
+      </LinkButton>
+      <LinkButton {...otherArgs} variant="tertiary">
+        Tertiary
+      </LinkButton>
+    </>
+  ),
+  decorators: [
+    (Story) => (
+      <div className="flex gap-8">
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export const LinkButtonVariantsReversed: Story = {
+  ...LinkButtonVariants,
+  parameters: {
+    reverseColors: true,
+  },
+}
+
+export const LinkButtonSizes: Story = {
+  render: (args) => (
+    <>
+      <LinkButton {...args} size="small" />
+      <LinkButton {...args} size="medium" />
+      <LinkButton {...args} size="large" />
```

Comment:
Suggestion:
```suggestion
      <LinkButton {...args} size="small">Small</LinkButton>
      <LinkButton {...args} size="medium">Medium</LinkButton>
      <LinkButton {...args} size="large">Large</LinkButton>
```

Reply:
Ah cheers, have updated to make that nicer for our consumers :)